core/tests/protractor.conf.js

```diff
@@ -56,6 +56,7 @@ exports.config = {
   // When run without a command line parameter, all suites will run. If run
   // with --suite=smoke, only the patterns matched by that suite will run.
   suites: {
+
```

Comment:
I did edit this file but I didn't added this file for commit my local repo, but this still got changed ! Does this mean if we edit file but dont commit or add this to staging area it will still be uploaded?